buildroot/bin/restore_configs

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
-git checkout Marlin/Configuration*.h 2>/dev/null
```

Comment:
`Configuration*.h` also matches `Configuration_Secure.h` which is created to define default WiFi settings. When it's present, `git checkout Marlin/Configuration*.h` fails, but `2>/dev/null` silences that failure and the whole `restore_configs` stops working seemingly for no reason.

Reply:
When I remove `2>/dev/null` from original `restore_configs` and run it with Configuration_Secure.h in place, I get:
```
error: pathspec 'Marlin/Configuration_Secure.h' did not match any file(s) known to git
```

So yeah, It's ignored for commiting, but also ignored to the point where it can't be checked-out by `git checkout`

Reply:
Is there a flag to `git checkout` that will suppress its complaint? Also, is `git checkout Marlin/Configuration*.h` being expanded by `git` or is it passing the pattern to something else and using the returned list?